### DISCUSSION TITLE
OKD 4.13 in fully disconnected deployment failed.

### DISCUSSION THREAD

**Discussion by kencafe** [okd-project/okd#1728] [Q&A] [answered]
Hi guys,
I deployed OKD 4.13.0-0.okd-2023-06-04-080300 with machine-os 38.20230514.3 Fedora CoreOS in fully disconnected environment.

The bootstrap node could not be up and the Podman on the bootstrap node ran etcdctl pod in loop. 
Bootstrap node had pull okd-content images from my registry (mirror-registry) successfully.

But bootstrap shows release-image-pivot service error.
Note: If the same configuration, if bootstrap node can connect to the internet, everything works well and I can deploy cluster completely. 
`[root@service ~]# ssh core@bootstrap
The authenticity of host 'bootstrap (10.0.10.3)' can't be established.
ED25519 key fingerprint is SHA256:3z50jTNW/rh/2JrAfkCx8te83S/NfFWz62pmP8zT9D8.
This key is not known by any other names
Are you sure you want to continue connecting (yes/no/[fingerprint])? yes
Warning: Permanently added 'bootstrap' (ED25519) to the list of known hosts.
This is the bootstrap node; it will be destroyed when the master is fully up.

The primary services are release-image.service followed by bootkube.service. To watch their status, run e.g.

  journalctl -b -f -u release-image.service -u bootkube.service
Fedora CoreOS 38.20230430.3.1
Tracker: https://github.com/coreos/fedora-coreos-tracker
Discuss: https://discussion.fedoraproject.org/tag/coreos

[systemd]
Failed Units: 1
  release-image-pivot.service`

`[root@bootstrap ~]# podman images
REPOSITORY                     TAG         IMAGE ID      CREATED       SIZE
quay.io/openshift/okd          <none>      ac5fa41e1ce9  3 months ago  429 MB
quay.io/openshift/okd-content  <none>      5c437aa03227  3 months ago  2.44 GB
quay.io/openshift/okd-content  <none>      211e2b5b26d7  3 months ago  637 MB
quay.io/openshift/okd-content  <none>      1f065111797f  3 months ago  617 MB
quay.io/openshift/okd-content  <none>      2dfe3217a83c  3 months ago  454 MB
quay.io/openshift/okd-content  <none>      54fe416377eb  3 months ago  591 MB
quay.io/openshift/okd-content  <none>      746cc81c7af1  3 months ago  459 MB
quay.io/openshift/okd-content  <none>      2b89a6fcb0ed  4 months ago  461 MB
quay.io/openshift/okd-content  <none>      63ebd824018e  4 months ago  461 MB
quay.io/openshift/okd-content  <none>      84a5416b0575  5 months ago  447 MB
quay.io/openshift/okd-content  <none>      9a33ef6046ef  5 months ago  440 MB
quay.io/openshift/okd-content  <none>      6c9aace496cc  5 months ago  428 MB`

`[root@bootstrap ~]# podman ps -a
CONTAINER ID  IMAGE                                                                                                  COMMAND               CREATED                 STATUS                 PORTS       NAMES
ae5b15b74f6f  quay.io/openshift/okd-content@sha256:619b53996ba758d0085a7414b021b52f97b7713ada266866cd664e8ed08672af  --dial-timeout=10...  Less than a second ago  Up Less than a second              etcdctl`

Please help me to find out why it is not working.

Many thanks

**Comment by vrutkovs** [comment]
Please attach [log bundle](https://docs.okd.io/latest/installing/installing-troubleshooting.html#installation-bootstrap-gather_installing-troubleshooting)

**Comment by kencafe** [comment]
Hi Vady,
This is gather logs I had manually collected from bootstrap:

1.bootkube.service service logs
`Sep 08 13:14:18 bootstrap bootkube.sh[123682]: Error: unhealthy cluster
Sep 08 13:14:18 bootstrap etcdctl[123693]: Error: unhealthy cluster
Sep 08 13:14:18 bootstrap podman[123682]: 2023-09-08 13:14:18.135088822 +0000 UTC m=+5.157845951 container died d4c2b5e1a87b4bc990abb60ed7a4af307d8b49924cf9ab903663d527c2122178 (image=quay.io/openshift/okd-content@sha256:619b53996ba758d0085a7414b021b52f97b7713ada266866cd664e8ed08672af, name=etcdctl, vcs-url=https://github.com/openshift/etcd, vcs-ref=f70da9d78221bc3e6bf8ac14c0c4ecc106f4f57d, summary=Provides the latest release of the Red Hat Extended Life Base Image., io.openshift.maintainer.component=Release, version=v4.13.0, io.openshift.build.commit.author=, io.openshift.maintainer.project=OCPBUGS, io.k8s.display-name=etcd server, io.openshift.ci.from.base=sha256:d8f186097a7de08a731a03eb8c9ea7a2823bb21256c8e25f365ba7172a81592f, maintainer=Sam Batschelet <sbatsche@redhat.com>, url=https://access.redhat.com/containers/#/registry.access.redhat.com/openshift/ose-base/images/v4.13.0-202303071354.p0.g42aa9c3.assembly.stream, io.openshift.tags=openshift,base, build-date=2023-03-07T20:27:22, vcs-type=git, com.redhat.license_terms=https://www.redhat.com/agreements, io.openshift.build.commit.date=, io.openshift.build.commit.id=f70da9d78221bc3e6bf8ac14c0c4ecc106f4f57d, io.openshift.build.commit.url=https://github.com/openshift/images/commit/42aa9c3e0e8c49358a74f9de8da8eca896c51892, io.openshift.build.name=, io.openshift.build.source-context-dir=, io.buildah.version=1.26.4, architecture=x86_64, release=202303071354.p0.g42aa9c3.assembly.stream, com.redhat.build-host=cpt-1001.osbs.prod.upshift.rdu2.redhat.com, io.openshift.expose-services=, description=This is the base image from which all OpenShift Container Platform images inherit., com.redhat.component=openshift-enterprise-base-container, io.openshift.build.namespace=, io.openshift.build.source-location=https://github.com/openshift/etcd, License=GPLv2+, io.k8s.description=etcd is a distributed key-value store which stores the persistent master state for Kubernetes and OpenShift., name=openshift/ose-base, vendor=Red Hat, Inc., io.openshift.build.commit.ref=openshift-4.13, io.openshift.build.commit.message=, distribution-scope=public)
Sep 08 13:14:18 bootstrap podman[123702]: 2023-09-08 13:14:18.217747593 +0000 UTC m=+0.068916181 container remove d4c2b5e1a87b4bc990abb60ed7a4af307d8b49924cf9ab903663d527c2122178 (image=quay.io/openshift/okd-content@sha256:619b53996ba758d0085a7414b021b52f97b7713ada266866cd664e8ed08672af, name=etcdctl, io.openshift.build.commit.url=https://github.com/openshift/images/commit/42aa9c3e0e8c49358a74f9de8da8eca896c51892, io.openshift.maintainer.project=OCPBUGS, release=202303071354.p0.g42aa9c3.assembly.stream, com.redhat.component=openshift-enterprise-base-container, io.openshift.expose-services=, com.redhat.license_terms=https://www.redhat.com/agreements, name=openshift/ose-base, io.k8s.display-name=etcd server, io.openshift.build.namespace=, io.openshift.build.commit.id=f70da9d78221bc3e6bf8ac14c0c4ecc106f4f57d, io.openshift.tags=openshift,base, vcs-ref=f70da9d78221bc3e6bf8ac14c0c4ecc106f4f57d, build-date=2023-03-07T20:27:22, io.openshift.build.source-context-dir=, vendor=Red Hat, Inc., io.openshift.build.commit.author=, io.openshift.build.source-location=https://github.com/openshift/etcd, maintainer=Sam Batschelet <sbatsche@redhat.com>, description=This is the base image from which all OpenShift Container Platform images inherit., architecture=x86_64, io.openshift.build.commit.message=, io.buildah.version=1.26.4, io.openshift.build.commit.ref=openshift-4.13, io.openshift.ci.from.base=sha256:d8f186097a7de08a731a03eb8c9ea7a2823bb21256c8e25f365ba7172a81592f, url=https://access.redhat.com/containers/#/registry.access.redhat.com/openshift/ose-base/images/v4.13.0-202303071354.p0.g42aa9c3.assembly.stream, com.redhat.build-host=cpt-1001.osbs.prod.upshift.rdu2.redhat.com, vcs-url=https://github.com/openshift/etcd, version=v4.13.0, distribution-scope=public, io.openshift.build.commit.date=, summary=Provides the latest release of the Red Hat Extended Life Base Image., License=GPLv2+, io.openshift.maintainer.component=Release, vcs-type=git, io.openshift.build.name=, io.k8s.description=etcd is a distributed key-value store which stores the persistent master state for Kubernetes and OpenShift.)
Sep 08 13:14:18 bootstrap bootkube.sh[3283]: etcdctl failed. Retrying in 5 seconds...
Sep 08 13:14:23 bootstrap podman[123846]: 2023-09-08 13:14:23.315433107 +0000 UTC m=+0.064256879 container create b26deaedecfe9196254f0b29fb9769602f76127e2c4eb7e74d57ae8e8a53a067 (image=quay.io/openshift/okd-content@sha256:619b53996ba758d0085a7414b021b52f97b7713ada266866cd664e8ed08672af, name=etcdctl, vcs-ref=f70da9d78221bc3e6bf8ac14c0c4ecc106f4f57d, url=https://access.redhat.com/containers/#/registry.access.redhat.com/openshift/ose-base/images/v4.13.0-202303071354.p0.g42aa9c3.assembly.stream, io.buildah.version=1.26.4, release=202303071354.p0.g42aa9c3.assembly.stream, vendor=Red Hat, Inc., io.openshift.expose-services=, io.openshift.build.commit.url=https://github.com/openshift/images/commit/42aa9c3e0e8c49358a74f9de8da8eca896c51892, distribution-scope=public, io.openshift.tags=openshift,base, architecture=x86_64, io.openshift.build.namespace=, io.openshift.build.name=, vcs-url=https://github.com/openshift/etcd, name=openshift/ose-base, io.openshift.build.commit.id=f70da9d78221bc3e6bf8ac14c0c4ecc106f4f57d, io.openshift.maintainer.project=OCPBUGS, description=This is the base image from which all OpenShift Container Platform images inherit., io.openshift.build.commit.ref=openshift-4.13, io.openshift.build.commit.date=, License=GPLv2+, io.k8s.display-name=etcd server, com.redhat.build-host=cpt-1001.osbs.prod.upshift.rdu2.redhat.com, io.openshift.build.source-context-dir=, com.redhat.component=openshift-enterprise-base-container, io.openshift.build.commit.author=, io.openshift.maintainer.component=Release, com.redhat.license_terms=https://www.redhat.com/agreements, vcs-type=git, maintainer=Sam Batschelet <sbatsche@redhat.com>, io.openshift.build.commit.message=, io.k8s.description=etcd is a distributed key-value store which stores the persistent master state for Kubernetes and OpenShift., summary=Provides the latest release of the Red Hat Extended Life Base Image., version=v4.13.0, io.openshift.ci.from.base=sha256:d8f186097a7de08a731a03eb8c9ea7a2823bb21256c8e25f365ba7172a81592f, io.openshift.build.source-location=https://github.com/openshift/etcd, build-date=2023-03-07T20:27:22)
Sep 08 13:14:23 bootstrap podman[123846]: 2023-09-08 13:14:23.297187783 +0000 UTC m=+0.046011543 image pull 6c9aace496cc88a80d7b56b7ac9942cb8940f95f9d9b567681892c3ef81673a5 quay.io/openshift/okd-content@sha256:619b53996ba758d0085a7414b021b52f97b7713ada266866cd664e8ed08672af
Sep 08 13:14:23 bootstrap podman[123846]: 2023-09-08 13:14:23.389037236 +0000 UTC m=+0.137861008 container init b26deaedecfe9196254f0b29fb9769602f76127e2c4eb7e74d57ae8e8a53a067 (image=quay.io/openshift/okd-content@sha256:619b53996ba758d0085a7414b021b52f97b7713ada266866cd664e8ed08672af, name=etcdctl, com.redhat.build-host=cpt-1001.osbs.prod.upshift.rdu2.redhat.com, io.openshift.build.commit.author=, distribution-scope=public, io.buildah.version=1.26.4, io.openshift.build.commit.ref=openshift-4.13, com.redhat.component=openshift-enterprise-base-container, description=This is the base image from which all OpenShift Container Platform images inherit., io.openshift.tags=openshift,base, version=v4.13.0, io.openshift.build.commit.id=f70da9d78221bc3e6bf8ac14c0c4ecc106f4f57d, vendor=Red Hat, Inc., name=openshift/ose-base, io.k8s.description=etcd is a distributed key-value store which stores the persistent master state for Kubernetes and OpenShift., io.openshift.expose-services=, io.k8s.display-name=etcd server, vcs-url=https://github.com/openshift/etcd, io.openshift.maintainer.component=Release, io.openshift.maintainer.project=OCPBUGS, summary=Provides the latest release of the Red Hat Extended Life Base Image., vcs-ref=f70da9d78221bc3e6bf8ac14c0c4ecc106f4f57d, License=GPLv2+, io.openshift.ci.from.base=sha256:d8f186097a7de08a731a03eb8c9ea7a2823bb21256c8e25f365ba7172a81592f, io.openshift.build.namespace=, vcs-type=git, maintainer=Sam Batschelet <sbatsche@redhat.com>, com.redhat.license_terms=https://www.redhat.com/agreements, release=202303071354.p0.g42aa9c3.assembly.stream, io.openshift.build.source-context-dir=, io.openshift.build.commit.date=, build-date=2023-03-07T20:27:22, url=https://access.redhat.com/containers/#/registry.access.redhat.com/openshift/ose-base/images/v4.13.0-202303071354.p0.g42aa9c3.assembly.stream, architecture=x86_64, io.openshift.build.name=, io.openshift.build.source-location=https://github.com/openshift/etcd, io.openshift.build.commit.url=https://github.com/openshift/images/commit/42aa9c3e0e8c49358a74f9de8da8eca896c51892, io.openshift.build.commit.message=)
Sep 08 13:14:23 bootstrap podman[123846]: 2023-09-08 13:14:23.39543529 +0000 UTC m=+0.144259064 container start b26deaedecfe9196254f0b29fb9769602f76127e2c4eb7e74d57ae8e8a53a067 (image=quay.io/openshift/okd-content@sha256:619b53996ba758d0085a7414b021b52f97b7713ada266866cd664e8ed08672af, name=etcdctl, architecture=x86_64, version=v4.13.0, io.openshift.build.commit.message=, io.openshift.build.commit.id=f70da9d78221bc3e6bf8ac14c0c4ecc106f4f57d, build-date=2023-03-07T20:27:22, io.buildah.version=1.26.4, com.redhat.license_terms=https://www.redhat.com/agreements, vcs-ref=f70da9d78221bc3e6bf8ac14c0c4ecc106f4f57d, release=202303071354.p0.g42aa9c3.assembly.stream, com.redhat.build-host=cpt-1001.osbs.prod.upshift.rdu2.redhat.com, io.openshift.build.namespace=, io.openshift.tags=openshift,base, io.openshift.ci.from.base=sha256:d8f186097a7de08a731a03eb8c9ea7a2823bb21256c8e25f365ba7172a81592f, distribution-scope=public, io.k8s.description=etcd is a distributed key-value store which stores the persistent master state for Kubernetes and OpenShift., io.openshift.build.commit.url=https://github.com/openshift/images/commit/42aa9c3e0e8c49358a74f9de8da8eca896c51892, io.openshift.build.commit.date=, io.openshift.build.source-location=https://github.com/openshift/etcd, io.openshift.maintainer.project=OCPBUGS, description=This is the base image from which all OpenShift Container Platform images inherit., io.openshift.expose-services=, url=https://access.redhat.com/containers/#/registry.access.redhat.com/openshift/ose-base/images/v4.13.0-202303071354.p0.g42aa9c3.assembly.stream, io.k8s.display-name=etcd server, name=openshift/ose-base, vendor=Red Hat, Inc., summary=Provides the latest release of the Red Hat Extended Life Base Image., vcs-url=https://github.com/openshift/etcd, maintainer=Sam Batschelet <sbatsche@redhat.com>, io.openshift.build.source-context-dir=, io.openshift.build.commit.ref=openshift-4.13, io.openshift.maintainer.component=Release, io.openshift.build.name=, com.redhat.component=openshift-enterprise-base-container, io.openshift.build.commit.author=, License=GPLv2+, vcs-type=git)
Sep 08 13:14:23 bootstrap podman[123846]: 2023-09-08 13:14:23.395911393 +0000 UTC m=+0.144735167 container attach b26deaedecfe9196254f0b29fb9769602f76127e2c4eb7e74d57ae8e8a53a067 (image=quay.io/openshift/okd-content@sha256:619b53996ba758d0085a7414b021b52f97b7713ada266866cd664e8ed08672af, name=etcdctl, version=v4.13.0, io.openshift.build.commit.date=, summary=Provides the latest release of the Red Hat Extended Life Base Image., vcs-ref=f70da9d78221bc3e6bf8ac14c0c4ecc106f4f57d, maintainer=Sam Batschelet <sbatsche@redhat.com>, io.openshift.build.name=, url=https://access.redhat.com/containers/#/registry.access.redhat.com/openshift/ose-base/images/v4.13.0-202303071354.p0.g42aa9c3.assembly.stream, io.k8s.display-name=etcd server, io.openshift.build.commit.message=, io.openshift.ci.from.base=sha256:d8f186097a7de08a731a03eb8c9ea7a2823bb21256c8e25f365ba7172a81592f, io.openshift.build.commit.author=, architecture=x86_64, License=GPLv2+, io.buildah.version=1.26.4, io.openshift.build.commit.id=f70da9d78221bc3e6bf8ac14c0c4ecc106f4f57d, io.openshift.expose-services=, io.openshift.tags=openshift,base, name=openshift/ose-base, io.openshift.build.commit.ref=openshift-4.13, io.openshift.build.source-context-dir=, build-date=2023-03-07T20:27:22, vcs-url=https://github.com/openshift/etcd, vcs-type=git, com.redhat.build-host=cpt-1001.osbs.prod.upshift.rdu2.redhat.com, io.openshift.maintainer.project=OCPBUGS, io.k8s.description=etcd is a distributed key-value store which stores the persistent master state for Kubernetes and OpenShift., com.redhat.license_terms=https://www.redhat.com/agreements, release=202303071354.p0.g42aa9c3.assembly.stream, com.redhat.component=openshift-enterprise-base-container, io.openshift.build.namespace=, distribution-scope=public, vendor=Red Hat, Inc., io.openshift.maintainer.component=Release, description=This is the base image from which all OpenShift Container Platform images inherit., io.openshift.build.commit.url=https://github.com/openshift/images/commit/42aa9c3e0e8c49358a74f9de8da8eca896c51892, io.openshift.build.source-location=https://github.com/openshift/etcd)
Sep 08 13:14:28 bootstrap etcdctl[123857]: {"level":"warn","ts":"2023-09-08T13:14:28.421Z","logger":"client","caller":"v3@v3.5.6/retry_interceptor.go:62","msg":"retrying of unary invoker failed","target":"etcd-endpoints://0xc00014e000/localhost:2379","attempt":0,"error":"rpc error: code = DeadlineExceeded desc = latest balancer error: last connection error: connection error: desc = \"transport: Error while dialing dial tcp [::1]:2379: connect: connection refused\""}
Sep 08 13:14:28 bootstrap etcdctl[123857]: https://localhost:2379 is unhealthy: failed to commit proposal: context deadline exceeded
Sep 08 13:14:28 bootstrap bootkube.sh[123846]: {"level":"warn","ts":"2023-09-08T13:14:28.421Z","logger":"client","caller":"v3@v3.5.6/retry_interceptor.go:62","msg":"retrying of unary invoker failed","target":"etcd-endpoints://0xc00014e000/localhost:2379","attempt":0,"error":"rpc error: code = DeadlineExceeded desc = latest balancer error: last connection error: connection error: desc = \"transport: Error while dialing dial tcp [::1]:2379: connect: connection refused\""}
Sep 08 13:14:28 bootstrap bootkube.sh[123846]: https://localhost:2379 is unhealthy: failed to commit proposal: context deadline exceeded
Sep 08 13:14:28 bootstrap bootkube.sh[123846]: Error: unhealthy cluster
Sep 08 13:14:28 bootstrap etcdctl[123857]: Error: unhealthy cluster`
2. podman logs
`{"level":"warn","ts":"2023-09-08T13:18:04.193Z","logger":"client","caller":"v3@v3.5.6/retry_interceptor.go:62","msg":"retrying of unary invoker failed","target":"etcd-endpoints://0xc00014e000/localhost:2379","attempt":0,"error":"rpc error: code = DeadlineExceeded desc = latest balancer error: last connection error: connection error: desc = \"transport: Error while dialing dial tcp [::1]:2379: connect: connection refused\""}
https://localhost:2379 is unhealthy: failed to commit proposal: context deadline exceeded
Error: unhealthy cluster`
3.container logs
`tail: cannot open '/var/lib/containers/storage/overlay-containers/*/userdata/ctr.log' for reading: No such file or directory`
4.kubelet.service and crio.service service logs
`{"level":"warn","ts":"2023-09-08T13:18:04.193Z","logger":"client","caller":"v3@v3.5.6/retry_interceptor.go:62","msg":"retrying of unary invoker failed","target":"etcd-endpoints://0xc00014e000/localhost:2379","attempt":0,"error":"rpc error: code = DeadlineExceeded desc = latest balancer error: last connection error: connection error: desc = \"transport: Error while dialing dial tcp [::1]:2379: connect: connection refused\""}
https://localhost:2379 is unhealthy: failed to commit proposal: context deadline exceeded
Error: unhealthy cluster
[root@bootstrap gather-logs-manual]# cat 3.container.log 
tail: cannot open '/var/lib/containers/storage/overlay-containers/*/userdata/ctr.log' for reading: No such file or directory
[root@bootstrap gather-logs-manual]# cat  4.kubelet-crio.log 
Sep 08 13:21:52 bootstrap podman[129901]: 2023-09-08 13:21:52.420842882 +0000 UTC m=+0.110002358 container start ec59e70796e0be18050c65c6ea1f7ffa367e162d289d033464312365ad81db16 (image=quay.io/openshift/okd@sha256:d696b82dbdac77fa98518ef1e8891104eb937d29c6461539407b301c79ea2177, name=laughing_pare, io.openshift.release=4.13.0-0.okd-2023-06-04-080300, io.openshift.release.base-image-digest=sha256:3725a071d12a08aacc3e9260d922db0f902057e996c4e1b5858de990a0899ee3)
Sep 08 13:21:52 bootstrap podman[129901]: 2023-09-08 13:21:52.420984442 +0000 UTC m=+0.110143976 container attach ec59e70796e0be18050c65c6ea1f7ffa367e162d289d033464312365ad81db16 (image=quay.io/openshift/okd@sha256:d696b82dbdac77fa98518ef1e8891104eb937d29c6461539407b301c79ea2177, name=laughing_pare, io.openshift.release=4.13.0-0.okd-2023-06-04-080300, io.openshift.release.base-image-digest=sha256:3725a071d12a08aacc3e9260d922db0f902057e996c4e1b5858de990a0899ee3)
Sep 08 13:21:52 bootstrap podman[129901]: 2023-09-08 13:21:52.345555042 +0000 UTC m=+0.034714556 image pull ac5fa41e1ce9c616cfe727071623a316eb6bacdb37b11d3a8efdf9fcebe61a00 quay.io/openshift/okd@sha256:d696b82dbdac77fa98518ef1e8891104eb937d29c6461539407b301c79ea2177
Sep 08 13:21:52 bootstrap laughing_pare[129912]: quay.io/openshift/okd-content@sha256:2b573b39a9196d509ce9117fee8dcb03dabc7195fbe20cf9ca473ef1b3a5a7bf
Sep 08 13:21:52 bootstrap podman[129901]: 2023-09-08 13:21:52.453104822 +0000 UTC m=+0.142264283 container died ec59e70796e0be18050c65c6ea1f7ffa367e162d289d033464312365ad81db16 (image=quay.io/openshift/okd@sha256:d696b82dbdac77fa98518ef1e8891104eb937d29c6461539407b301c79ea2177, name=laughing_pare, io.openshift.release.base-image-digest=sha256:3725a071d12a08aacc3e9260d922db0f902057e996c4e1b5858de990a0899ee3, io.openshift.release=4.13.0-0.okd-2023-06-04-080300)
Sep 08 13:21:52 bootstrap podman[129922]: 2023-09-08 13:21:52.530498962 +0000 UTC m=+0.064768585 container remove ec59e70796e0be18050c65c6ea1f7ffa367e162d289d033464312365ad81db16 (image=quay.io/openshift/okd@sha256:d696b82dbdac77fa98518ef1e8891104eb937d29c6461539407b301c79ea2177, name=laughing_pare, io.openshift.release=4.13.0-0.okd-2023-06-04-080300, io.openshift.release.base-image-digest=sha256:3725a071d12a08aacc3e9260d922db0f902057e996c4e1b5858de990a0899ee3)
Sep 08 13:21:52 bootstrap kubelet.sh[129951]: /usr/local/bin/kubelet.sh: line 7: /usr/bin/hyperkube: No such file or directory
Sep 08 13:21:52 bootstrap systemd[1]: kubelet.service: Main process exited, code=exited, status=127/n/a
Sep 08 13:21:52 bootstrap systemd[1]: kubelet.service: Failed with result 'exit-code'.
Sep 08 13:21:52 bootstrap systemd[1]: Failed to start kubelet.service - Kubernetes Kubelet.`
5.worker host container logs 
`tail: cannot open '/var/log/containers/*' for reading: No such file or directory
tail: no files remaining`

There is only etcdctl pod is running in looping.

**Reply by vrutkovs** [reply]
>But bootstrap shows release-image-pivot service error.

Which error? Could you attach the archive collected with `openshift-install gather bootstrap` instead of pasting it here?

**Comment by kencafe** [comment]
This is my logs bundle from the bootstrap host.

[log-bundle-20230908135858.tar.gz](https://github.com/okd-project/okd/files/12560633/log-bundle-20230908135858.tar.gz)

**Reply by vrutkovs** [reply, marked as answer]
```
Sep 08 10:20:03 bootstrap rpm-ostree[2967]: Txn Rebase on /org/projectatomic/rpmostree1/fedora_coreos failed: Creating importer: Failed to invoke skopeo proxy method OpenImage: remote error: (Mirrors also failed: [ac-registry.staging.xplat.online:8443/ocp4/openshift4@sha256:a70ed62bcd842bd75806638a64afeac3e691922566e7f0c312c60c80845a4953: getting username and password: 1 error occurred:
                                                    * reading JSON file "/run/containers/62011/auth.json": open /run/containers/62011/auth.json: permission denied
                                            
                                            ]): quay.io/openshift/okd-content@sha256:a70ed62bcd842bd75806638a64afeac3e691922566e7f0c312c60c80845a4953: pinging container registry quay.io: Get "https://quay.io/v2/": dial tcp: lookup quay.io: no such host
```

see #1500

**Reply by kencafe** [reply]
Had resolved in 1/2 minute. Many many thanks Vady.
